### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: include/ed/plugin.h [hunk 1]
```diff
@@ -32,13 +32,13 @@ class Plugin
...
+    virtual void configure(tue::Configuration /*config*/) {}
     virtual void initialize() {}
-    virtual void process(const WorldModel& world, UpdateRequest& req) {}
+    virtual void process(const WorldModel& /*world*/, UpdateRequest& /*req*/) {}
 
     // New
-    virtual void initialize(InitData& init) {}
-    virtual void process(const PluginInput& data, UpdateRequest& req) {}
+    virtual void initialize(InitData& /*init*/) {}
+    virtual void process(const PluginInput& /*data*/, UpdateRequest& /*req*/) {}
```
Comment (ar13pit) [inline include/ed/plugin.h:41]: Why keep those comments?

**Comment by ar13pit** [comment]
Also with the raw pointers, can they not be converted to smart pointers?

File: include/ed/plugin.h (hunk 1 above)
Comment (MatthijsBurgh) [reply include/ed/plugin.h:41]: A bit as a guideline...

Comment (ar13pit) [reply include/ed/plugin.h:41]: Looks weird. But if you want to keep it, I'm okay with it

**Comment by MatthijsBurgh** [comment]
Yep pointers can probably be converted to smart pointers. Future work ;)

**Pull request merged**